workers/unity/Assets/Fps/Scripts/GameLogic/RandomNameGenerator.cs

```diff
@@ -0,0 +1,70 @@
+﻿using UnityEngine;
+
+namespace Fps
+{
+    public static class RandomNameGenerator
+    {
+        public static string GetName()
+        {
+            var prefix = Prefixes[Random.Range(0, Prefixes.Length)];
+            var postfix = Postfixes[Random.Range(0, Postfixes.Length)];
+
+            var timeOut = 200;
```

Comment:
`timeOut` -> `attemptsLeft`

Reply:
👍 I'm all for better naming, but I'll leave it as-is because this class should be deprecated soon.